packages/dropdowns/src/Autocomplete/Autocomplete.tsx

```diff
@@ -48,7 +60,7 @@ const Autocomplete = ({ children, ...props }) => {
   const selectProps = getToggleButtonProps({
     onKeyDown: e => {
       if (isOpen) {
-        e.nativeEvent.preventDownshiftDefault = true;
+        (e.nativeEvent as any).preventDownshiftDefault = true;
```

Comment:
To me non-null operator would make more sense here than the extra parentheses.

```
e.nativeEvent!.preventDownshiftDefault = true;
```

That's the same as above right?

Reply:
This goes beyond the non-null operator though.

`preventDownshiftDefault` is a custom property that Downshift pony-fills to their event handling. Our render-prop wrappings don't show the correct types for this event and `any` is the easiest fix.